components/org.wso2.carbon.identity.api.user.session/org.wso2.carbon.identity.api.user.session.v1/src/main/resources/session.yaml

```diff
@@ -329,14 +386,8 @@ parameters:
 securityDefinitions:
   BasicAuth:
     type: basic
-  # ClientCertificate:
-  #   type: custom
   OAuth2:
     type: oauth2
     flow: accessCode
     authorizationUrl: https://localhost:9443/oauth/authorize
-    tokenUrl: https://localhost:9443/oauth/token
-    scopes:
-      read: Grants read access
-      write: Grants write access
-      admin: Grants read and write access to administrative information
+    tokenUrl: https://localhost:9443/oauth/token
```

Comment:
new line at the end?
authorizationUrl: https://localhost:9443/oauth2/authorize 
tokenUrl: https://localhost:9443/oauth2/token

Reply:
updated